README.md

```diff
@@ -38,6 +39,7 @@ That's it, you're done ... almost.
 |   80 |        80 | nginx   |
 | 2003 |      2003 | carbon  |
 | 8125 |      8125 | statsd  |
+| 8216 |      8126 | admin   |
```

Comment:
Looks like typo here `8216` should read `8126`. Other than that, this looks great.